File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.c
#include <stdio.h>
main(){
    int N;
    scanf("%d", &N);
    int arr[N];
    int x;
    scanf("%d", &x);
    
    for(int i=0; i<N; i++){
        scanf("%d", &arr[i]);
    }
    for(int j=0; j<N; j++){
        if(arr[j]<x){
            printf("%d", arr[j]);
            printf(" ");
        }
    }
}

File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.java
import java.util.Scanner;
class Main{
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        
        int N = sc.nextInt();
        int arr[] = new int[N];
        int x = sc.nextInt();
        
        for(int i=0; i<N; i++){
            arr[i] = sc.nextInt();             
        }
        for(int j=0; j<arr.length; j++){
            if(arr[j]<x){
                System.out.print(arr[j] + " ");
            }
        }
        
       
    }
}